docs/editor_manual/new_pages/inserting_images.rst

```diff
@@ -1,6 +1,6 @@
 .. _inserting_images:
 
-Inserting images and videos in a page
+Inserting images in a page
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Please can you reduce the length of the underline to match the text? RestructuredText is picky about this...

Reply:
Done!